src/Codeception/Step/Comment.php

```diff
@@ -1,37 +1,41 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Step;
 
 use Codeception\Lib\ModuleContainer;
 use Codeception\Step as CodeceptionStep;
+use function mb_strcut;
 
 class Comment extends CodeceptionStep
 {
-    public function __toString()
+    public function __toString(): string
     {
         return (string) $this->getAction();
     }
 
-    public function toString($maxLength)
+    public function toString($maxLength): string
     {
-        return mb_strcut($this->__toString(), 0, $maxLength, 'utf-8');
+        return mb_strcut((string) $this, 0, $maxLength, 'utf-8');
     }
 
-    public function getHtml($highlightColor = '#732E81')
+    public function getHtml($highlightColor = '#732E81'): string
```

Comment:
It is odd that unused argument has default value.

Reply:
some tests assume that this default value exists